tests/blockchain/test_blockchain_transactions.py

```diff
@@ -161,16 +161,15 @@ async def test_validate_blockchain_duplicate_output(self, two_nodes):
             if coin.puzzle_hash == coinbase_puzzlehash:
                 spend_coin = coin
 
-        spend_bundle = wallet_a.generate_signed_transaction(1000, receiver_puzzlehash, spend_coin)
-        spend_bundle_double = wallet_a.generate_signed_transaction(1000, receiver_puzzlehash, spend_coin)
-
-        block_spendbundle = SpendBundle.aggregate([spend_bundle, spend_bundle_double])
+        spend_bundle = wallet_a.generate_signed_transaction(
+            1000, receiver_puzzlehash, spend_coin, additional_outputs=[(receiver_puzzlehash, 1000)]
+        )
```

Comment:
this is the fix. Create a single spend bundle that creates two identical outputs. Previously the two spend-bundles themselves were identical also, attempting to spend the same coin twice.